app/assets/stylesheets/limber/screen.scss

```
@@ -526,6 +520,26 @@ form + .btn-block {
   }
 }
 
+// Stackable cards, designed to be positioned directly on
+// top of each other to show steps in a process
```

Could I suggest a rewording to avoid ambiguity?  Directly on top of each other initially means overlaying each other like a deck of cards to me.

```suggestion
// Stackable cards, designed to be place in a vertical
// stack, illustrating steps in a process.
```